pkgs/development/python-modules/gentools/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage, lib, fetchFromGitHub, pytest
+, typing, funcsigs
+, isPy27, isPy33, isPy34
```

Comment:
```suggestion
```
Be sure to remove these after the below is applied and add `pythonOlder`.

Reply:
Ops, shouldn't commit early in the morning ^_^